src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -702,7 +702,7 @@ private void testDownloadAction() {
                                               .withUserId("CFResultsUiT.instr");
         browser.driver.get(reportUrl.toAbsoluteString());
         String afterReportDownloadUrl = browser.driver.getCurrentUrl();
-        assertFalse(reportUrl.equals(afterReportDownloadUrl));
+        assertFalse(reportUrl.toString().equals(afterReportDownloadUrl));
```

Comment:
Hi @AlexM9200 , nice work! I know it's not related to your issue, but you still can improve this line to use assertEquals instead of assertFalse here:
`assertEquals(reportUrl.toString(), afterReportDownloadUrl)`